### DISCUSSION TITLE
Support for vue-grid-layout

### DISCUSSION THREAD

**Discussion by Wzhipeng** [zauberzeug/nicegui#1546] [Ideas / Feature Requests]
### Description

I want to use the vue-grid-layout component in nicegui, but I don't know how to add this component to the project.  Is there a way for nicegui to add other vue components?

**Comment by Wzhipeng** [comment]
when I use the vue structure in custom_vue_component, I cannot import it and I cannot define the component.

**Comment by rodja** [comment]
It would be great if we could support the vue-grid-layout. If we solve #1454 this should be fairly simple.
